imports/plugins/core/payments/server/no-meteor/mutations/enablePaymentMethodForShop.js

```diff
@@ -0,0 +1,36 @@
+import ReactionError from "@reactioncommerce/reaction-error";
+import SimpleSchema from "simpl-schema";
+import { Shops } from "/lib/collections";
+import { paymentMethods as allPaymentMethods } from "/imports/plugins/core/core/server/no-meteor/pluginRegistration";
+
+const paramsSchema = new SimpleSchema({
+  isEnabled: Boolean,
+  paymentMethodName: String,
+  shopId: String
+});
+
+export default async function enablePaymentMethodForShop(context, input = {}) {
+  paramsSchema.validate(input);
+  const { isEnabled, paymentMethodName, shopId } = input;
+
+  if (!allPaymentMethods[paymentMethodName]) {
+    throw new ReactionError("not-found", "Requested payment method is invalid");
+  }
+
+  const shop = await context.queries.shopById(context, shopId);
```

Comment:
`shop` might be undefined. If so, need to throw a not-found for that same as above.